eZ/Publish/Core/Persistence/Legacy/Tests/Content/Gateway/DoctrineDatabaseTest.php

```diff
@@ -2116,4 +2116,28 @@ private function assertContentVersionAttributesLanguages(
                 ->orderBy('id')
         );
     }
+
+    /**
+     * @throws \Doctrine\DBAL\Driver\Exception
+     * @throws \Doctrine\DBAL\DBALException
+     * @throws \Doctrine\DBAL\Exception
+     */
+    public function testLoadVersionInfoList(): void
+    {
+        $this->insertDatabaseFixture(
+            __DIR__ . '/../_fixtures/contentobjects.php'
+        );
+
+        $gateway = $this->getDatabaseGateway();
+
+        $results = $gateway->loadVersionInfoList([11]);
+
+        $orig = include __DIR__ . '/../_fixtures/extract_version_info_from_rows_multiple_versions.php';
+
+        $this->assertEquals(
+            [$orig[1]],
+            $results,
+            'Fixtures differ between what was previously stored(expected) and what it now generates(actual), this hints either some mistake in impl or that the fixture (../_fixtures/extract_content_from_rows_multiple_versions.php) and tests needs to be adapted.'
+        );
+    }
```

Comment:
This test doesn't bring enough value to keep it. It's redundant with an integration one, but actually tests less, because it's ran only against the one database engine we don't use - `sqlite`. Unit tests should never do any integration testing. Tests here are a big issue, we want actually to remove them once we make sure for each case there's corresponding proper integration coverage.

Additionally, if you see CI failures - there's a bug in this code reported by SonarCloud.

Reply:
Regarding this bug, changing it to `include_once` resulted in some other exceptions, so I kept it as in the methods above. Nevertheless, I remove this method.